internal/helm/helm.go

```diff
@@ -321,7 +321,7 @@ func Deploy(logger logr.Logger, parameters ChartParameters) error {
 	}
 
 	helmChart := appChart.HelmChart
-	helmVersion := ""
+	helmVersion := appChart.ChartVersion
 
```

Comment:
How do you see this interacting with https://github.com/epinio/epinio/blob/63028b0c605c10784188b8fe13a9f486caa90103/internal/helm/helm.go#L336-L340 just a few lines below ?

IOW when the chart reference is to a helm repository then the version is taking from the chart reference, with chart and version separated by `:`.

Could something similar be done with OCI references ? Assuming that OCI references can be detected as such generally.